app/test_main.py

```diff
@@ -1,3 +1,52 @@
+from __future__ import annotations
+
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatDogYears:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,result",
+        [
+            (-1, -50, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ]
+    )
+    def test_human_age_is_correct(
+            self,
+            cat_age: int,
+            dog_age: int,
+            result: list
+    ) -> None:
+
+        assert get_human_age(cat_age, dog_age) == result
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_error",
+        [
+            (
+                "one",
+                "twenty",
+                TypeError),
```

Comment:
Mind that brackets should be on the same level
Also follow the same code style throughout the project. It's better to split on lines here the same as in previous @pytest.mark.parametrize decorator